README.md

```diff
@@ -9,25 +9,43 @@ The following steps will install the `tigerbeetle-node` module to your current w
 * NodeJS >= `14.0.0`. _(If the correct version is not installed, an installation error will occur)_ 
 
 > Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
-     
+
 ### YARN Package Manager
 
-```sh
-# Run the following from this directory
-yarn add
+```shell
+# Run the following from this directory:
+yarn set version latest
```

Comment:
Running this command modifies `.yarnrc.yml` as well as files in `.yarn` which are both tracked by git.

Are you sure this is something users should run? It seems like it would be more robust to have them stick to the version we have committed in the repository instead of the latest version available.